libs/SalesforceReact/SalesforceReact/Classes/SFOauthReactBridge.m

```diff
@@ -50,11 +49,9 @@ @implementation SFOauthReactBridge
 
 RCT_EXPORT_METHOD(logoutCurrentUser:(NSDictionary *)args callback:(RCTResponseSenderBlock)callback)
 {
-    __weak typeof(self) weakSelf = self;
     [SFSDKReactLogger d:[self class] format:@"logoutCurrentUser: arguments: %@", args];
     dispatch_async(dispatch_get_main_queue(), ^{
-         __strong typeof(weakSelf) strongSelf = weakSelf;
-        [strongSelf logout];
+        [[SFUserAccountManager sharedInstance] logout];
```

Comment:
Remove all convenience methods login/logout which were deciding between the use of SFAuthenticationManager & SFUserAccountManager.